packages/playwright-core/src/client/elementHandle.ts

```diff
@@ -20,10 +20,10 @@ import { promisify } from 'util';
 import { Frame } from './frame';
 import { JSHandle, parseResult, serializeArgument } from './jsHandle';
 import { assert } from '../utils/isomorphic/debug';
-import { fileUploadSizeLimit, mkdirIfNeeded } from '../common/fileUtils';
+import { fileUploadSizeLimit, mkdirIfNeeded } from './fileUtils';
 import { isString } from '../utils/isomorphic/rtti';
-import { mime } from '../utilsBundle';
```

Comment:
Let's drop `mime` from the dependencies then?

Reply:
I'll follow up.